AwsCryptographyPrimitives/runtimes/python/src/aws_cryptography_primitives/internaldafny/extern/ECDH.py

```diff
@@ -0,0 +1,394 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from enum import Enum
+
+import standard_library.internaldafny.generated.Wrappers as Wrappers
+from aws_cryptography_primitives.internaldafny.generated.ECDH import default__
+from aws_cryptography_primitives.smithygenerated.aws_cryptography_primitives.errors import (
+    AwsCryptographicPrimitivesError,
+    _smithy_error_to_dafny_error
+)
+
+import _dafny
+import cryptography.hazmat.primitives.asymmetric.ec
+from aws_cryptography_primitives.internaldafny.extern.Signature import (
+    _ecc_decode_compressed_point,
+    _ecc_encode_compressed_point_public_key,
+)
+
+from aws_cryptography_primitives.internaldafny.generated.ECDH import \
+    EccKeyPair_EccKeyPair
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePublicKey
+from cryptography.hazmat.primitives.serialization import (Encoding,
+                                                          NoEncryption,
+                                                          PrivateFormat,
+                                                          PublicFormat,
+                                                          load_der_public_key,
+                                                          load_pem_private_key)
+
+from aws_cryptography_primitives.internaldafny.extern.Signature import \
+    _ECC_CURVE_PARAMETERS
+
+CreateExternDerivesharedSecretSuccess = default__.CreateExternDerivesharedSecretSuccess
+CreateExternDerivesharedSecretError = default__.CreateExternDerivesharedSecretError
+CreateExternEccKeyGenFailure = default__.CreateExternEccKeyGenFailure
+CreateExternEccKeyGenSuccess = default__.CreateExternEccKeyGenSuccess
+CreateExternParsePublicKeySuccess = default__.CreateExternParsePublicKeySuccess
+CreateExternParsePublicKeyError = default__.CreateExternParsePublicKeyError
+CreateExternGetPublicKeyFromPrivateError = default__.CreateExternGetPublicKeyFromPrivateError
+CreateExternValidatePublicKeyError = default__.CreateExternValidatePublicKeyError
+CreateExternValidatePublicKeySuccess = default__.CreateExternValidatePublicKeySuccess
+CreateExternCompressPublicKeySuccess = default__.CreateExternCompressPublicKeySuccess
+CreateExternCompressPublicKeyError = default__.CreateExternCompressPublicKeyError
+CreateExternDecompressPublicKeyError = default__.CreateExternDecompressPublicKeyError
+CreateExternDecompressPublicKeySuccess = default__.CreateExternDecompressPublicKeySuccess
```

Comment:
hmm, would it be appropriate python styling if we were to move all  these Error/Success object and place them in a different extern file called "CreateSuccessErrors.py"? this way all externs could you just 
`from CreateSuccessErrors import *`?

Reply:
Sure, I think that's more complicated though. If you don't like this I'd prefer to just call (ex) `default__. CreateExternDerivesharedSecretSuccess` from directly within code